pom.xml

```diff
@@ -116,62 +89,18 @@
       <dependency>
         <groupId>com.github.dbmdz.flusswerk</groupId>
         <artifactId>framework</artifactId>
-        <version>${project.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.assertj</groupId>
-        <artifactId>assertj-core</artifactId>
-        <version>${version.assertj}</version>
-        <scope>test</scope>
-      </dependency>
-      <dependency>
-        <groupId>javax.annotation</groupId>
-        <artifactId>javax.annotation-api</artifactId>
-        <version>${version.javax.annotation-api}</version>
-      </dependency>
-      <dependency>
-        <groupId>javax.validation</groupId>
-        <artifactId>validation-api</artifactId>
-        <version>${version.validation-api}</version>
+        <version>${version.flusswerk}</version>
```

Comment:
Wieso wird hier nicht die project.version verwendet?

Reply:
Das geht nicht, weil `project.version` in einem Workflow-Job zu dessen Version auflösen würde (das Projekt ist ja der Workflow-Job). Das zerschießt dann das Dependency-Management zu Flusswerk.